corehq/apps/users/models.py

```diff
@@ -2124,6 +2124,18 @@ def get_sql_location(self, domain):
         if loc_id:
             return SQLLocation.objects.get_or_None(domain=domain, location_id=loc_id)
 
+    def get_assigned_location_ids(self, domain):
+        return getattr(self.get_domain_membership(domain), 'assigned_location_ids', None)
+
+    @memoized
+    def get_assigned_sql_locations(self, domain):
+        from corehq.apps.locations.models import SQLLocation
+        loc_ids = self.get_assigned_location_ids(domain)
+        if loc_ids:
+            return SQLLocation.objects.get_locations(loc_ids)
+        else:
+            return []
+
```

Comment:
Housa!!! 
Great minds! Absolutely :)

Can i keep this version? For the naming using `assigned` and It reusing already defined methods like `get_locations`?